cmd/buildah/push.go

```diff
@@ -76,6 +79,7 @@ func init() {
 	flags.BoolVarP(&opts.disableCompression, "disable-compression", "D", false, "don't compress layers")
 	flags.StringVarP(&opts.format, "format", "f", "", "manifest type (oci, v2s1, or v2s2) to use when saving image using the 'dir:' transport (default is manifest type of source)")
 	flags.BoolVarP(&opts.quiet, "quiet", "q", false, "don't output progress information when pushing images")
+	flags.BoolVar(&opts.purge, "purge", false, "remove the manifest list if push succeeds")
```

Comment:
Why `purge` and not `rm`?

Reply:
Because manifest used --purge.  Moved both to use --rm, and aliases purge to rm.